src/transformers/generation/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
It's a bit sad to have an empty init here. Would probably will make it easier for users to have everything importable in this submodule directly? That would just quire building this init like the main init of the models init.

Reply:
Sounds good 👍